hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/AddUpdateRolloutWindowLayout.java

```diff
@@ -622,42 +600,38 @@ private TextField createRolloutNameField() {
         return rolloutNameField;
     }
 
-    private Label createMandatoryLabel() {
-        final Label madatoryLbl = new Label(i18n.get("label.mandatory.field"));
-        madatoryLbl.addStyleName(SPUIStyleDefinitions.SP_TEXTFIELD_ERROR + " " + ValoTheme.LABEL_SMALL);
-        return madatoryLbl;
-    }
-
     private String getRolloutName() {
         return HawkbitCommonUtil.trimAndNullIfEmpty(rolloutName.getValue());
     }
 
-    private DistributionSetIdName getDistributionSetSelected() {
-        return (DistributionSetIdName) distributionSet.getValue();
-    }
-
     class ErrorThresoldOptionValidator implements Validator {
         private static final long serialVersionUID = 9049939751976326550L;
 
         @Override
         public void validate(final Object value) {
             try {
                 if (HawkbitCommonUtil.trimAndNullIfEmpty(noOfGroups.getValue()) == null
-                        || HawkbitCommonUtil.trimAndNullIfEmpty((String) targetFilterQueryCombo.getValue()) == null) {
+                        || (HawkbitCommonUtil.trimAndNullIfEmpty((String) targetFilterQueryCombo.getValue()) == null
```

Comment:
The condition is hard to understand. Please extract a method and provide a proper name